### PR TITLE
Update postman collection with new filter operations_app endpoint

### PR DESCRIPTION
**Pull request by danishjamal104**

### CODE REVIEW

**Comment by danishjamal104** [comment]
@avikganguly01 and @SubhamPramanik please review and merge. 

Screenshot 2022-04-24 at 5 30 28 PM

**Comment by fynmanoj** [comment]
Can you squash 3 commits into 1 ?

File: PostmanCollections/Payment Hub.json [hunk 1]
```diff
@@ -1531,6 +1531,353 @@
...
+								"header": [
+									{
+										"key": "Platform-TenantId",
+										"value": "oaf",
+										"type": "text"
+									}
+								],
+								"body": {
+									"mode": "raw",
+									"raw": "[\n    \"70830851a2f3rPVoWj3U\"\n]",
```
Comment (fynmanoj) [inline PostmanCollections/Payment Hub.json:1588]: What is this ID ? 70830851a2f3rPVoWj3U ?

**Review by fynmanoj — APPROVED** [review]
LGTM

**Pull request merged**